draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -197,30 +199,34 @@ Nameservers MUST accept catalog zones even with those labels not really unique;
 Having a large number of member zones in a single RRset may cause the RRset to be too large to be conveyed via DNS messages which make up a zone transfer.
 Having the zones uniquely tagged with the `<m-unique-N>` label ensures the list of member zones can be split over multiple DNS messages in a zone transfer.
 
-The `<m-unique-N>` label also enables the state for a zone to be reset. (see (#zonereset))
+The `<m-unique-N>` label also enables the state for a zone to be reset (see (#zonereset)).
 As long as no zone state needs to be reset at the authoritative nameservers, the unique label associated with a zone SHOULD remain the same.
 
 The CLASS field of every RR in a catalog zone MUST be IN (1).
 
+[FIXME: how about turning this around? Because they are not for querying, queries to it should be discouraged by using a huge TTL]
```

Comment:
It would certainly make querying more difficult :)
How about just removing "and therefore a value of zero (0) is ￼RECOMMENDED." bit

Reply:
Makes sense.